features/step_definitions/assessment_steps.rb

```diff
@@ -4,12 +4,14 @@
 
 When('I choose to schedule an assessment') do
   @assessment_type = 'schedule'
-  click_link 'Schedule an assessment'
+  page.find('#assessmentDropdownElements').click
```

Comment:
This breaks if you run in a JS browser, e.g. firefox
Have pushed up https://github.com/wearefuturegov/beacon/tree/fix/assessment_tests_browser
We can leave it until after this release as it's not critical and the tests are passing in the pipeline

Reply:
sorted